src/app/booking/OOSPlannerSection.js

```diff
@@ -260,80 +406,63 @@ function FamilyBookings() {
-
-            <StyledTableCell colSpan={5}>
-              <Box sx={{ display: "flex", justifyContent: "space-between", alignItems: "center" }}>
-                <Typography variant="h6">{selectedDate.format("MMMM YYYY")}</Typography>
-                <Box sx={{ display: "flex", alignItems: "center", gap: 1 }}>
-                  <IconButton onClick={handlePreviosWeek}>
-                    <ArrowBackIosNewIcon />
-                  </IconButton>
-                  <Button color="grey" sx={{ py: 0.2 }} onClick={() => setSelectedDate(dayjs())}>
-                    Today
+    !isLoadingChildren && !isLoadingBookings &&
+      <TableContainer component={Box}>
+        <Table sx={{ minWidth: 650 }} aria-label="simple table">
+          <TableHead>
+            <TableRow>
+              <StyledTableCell rowSpan={2} sx={{ verticalAlign: "bottom", textAlign: "center" }}>
+                <FamilyRestroomIcon />
+                <Typography sx={{ fontWeight: 700 }}>Family</Typography>
+              </StyledTableCell>
+
+              <StyledTableCell colSpan={5}>
+                <Box sx={{ display: "flex", justifyContent: "space-between", alignItems: "center" }}>
+                  <Typography variant="h6">{selectedDate.format("MMMM YYYY")}</Typography>
+                  <Box sx={{ display: "flex", alignItems: "center", gap: 1 }}>
+                    <IconButton onClick={handlePreviosWeek}>
+                      <ArrowBackIosNewIcon />
+                    </IconButton>
+                    <Button color="grey" sx={{ py: 0.2 }} onClick={() => setSelectedDate(dayjs())}>
+                      Today
+                    </Button>
+                    <IconButton onClick={handleNextWeek}>
+                      <ArrowForwardIosIcon />
+                    </IconButton>
+                  </Box>
+                  <Button startIcon={<IosShareIcon />} variant="outlined" color="grey">
+                    Share Calendar
                   </Button>
-                  <IconButton onClick={handleNextWeek}>
-                    <ArrowForwardIosIcon />
-                  </IconButton>
                 </Box>
-                <Button startIcon={<IosShareIcon />} variant="outlined" color="grey">
-                  Share Calendar
+              </StyledTableCell>
+            </TableRow>
+
+            <TableRow sx={{ borderBottom: "1px solid", borderColor: "grey.300" }}>
+              {weekDates.map((date, index) => (
+                <StyledHeaderTableCell key={index} align="center">
+                  <Box
+                    sx={{ bgcolor: dayjs().isSame(date, "day") ? "grey.100" : "transparent", borderRadius: 1.5, py: 1 }}
+                  >
+                    {formatDate(date)}
+                  </Box>
+                </StyledHeaderTableCell>
+              ))}
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            <FamilyBookings childrenData={children} bookings={bookings} weekDates={weekDates} />
```

Comment:
Children bookings are binded to a FamilyBookings component, so I suppose they should be retrieved in that component the same way as you do for FriendsBookings. Children are common for them, so ok for children to live here.

Reply:
Child bookings are used for unpaid bookings. If I move receiving bookings to the FamilyBookings component, I'll need to move the "Pay Now" button and all the logic around it, but I think it should be in the BookingTable component

Reply:
I'd move pay now to family bookings. It is not a part of bookings table, yes. But now we keep a state for FamilyBookings in a parent component, which only passes it down. That looks a bigger violation that moving pay now to FamilyBookings, taking into account, that PayNow is also works with FamilyBookings.